operator/api/auto/v1alpha1/workspace_types.go

```diff
@@ -101,6 +101,13 @@ type WorkspaceSpec struct {
 	// +optional
 	PodTemplate *EmbeddedPodTemplateSpec `json:"podTemplate,omitempty"`
 
+	// PulumiLogVerbosity is the log verbosity level to use for the Pulumi CLI.
+	// If unset,verbose logging is disabled.
+	// See: https://www.pulumi.com/docs/iac/support/troubleshooting/#verbose-logging
+	// for more information about log levels.
+	// +optional
+	PulumiLogVerbosity uint32 `json:"pulumiLogLevel,omitempty"`
```

Comment:
I think i would advocate for `logLevel` rather than `pulumiLogLevel` because the workspace is very much pulumi-specific.

Reply:
True, but we also have verbosity for the agent/server itself. I'd rather have more specific names to avoid ambiguity.

Reply:
Note: the agent cmd supports `verbose` currently enables/disables verbose logging for the agent binary. Having `logLevel` instead of `pulumiLogLevel` may confuse users into thinking that the logLevel is for the agent binary itself, instead of the Pulumi CLI.